apps/client/src/app/career-qna/page.tsx

```diff
@@ -0,0 +1,87 @@
+'use client';
+
+import styled from '@emotion/styled';
+import { Flex, Stack, Text } from '@sickgyun/ui';
+import { useState } from 'react';
+import Footer from '@/components/common/Footer';
+import Header from '@/components/common/Header';
+import QnaAskBox from '@/components/qna-posting/QnaAskBox';
+import QnaListBox from '@/components/qna-posting/QnaListBox';
+import QnaPostingList from '@/components/qna-posting/QnaPostingList';
+import { QNA_LIST } from '@/constants/qna-list';
+
+const CareerQna = () => {
+  const [currentQnaPage, setCurrentQnaPage] = useState(0);
+
+  const handlePrevCareerQna = () => {
+    setCurrentQnaPage((prevQna) => (prevQna > 0 ? prevQna - 1 : 0));
+  };
```

Comment:
```suggestion
  const handlePrevCareerQna = () => {
    if(currentQnaPage > 0) {
        setCurrentQnaPage((prev) => prev - 1);
    }
  };
```

이런식으로 짜는 것이 가독성이 더 좋을것같아용